test/ParameterStoreBuildWrapperPluginTest.groovy

```diff
@@ -25,6 +26,46 @@ class ParameterStoreBuildWrapperPluginTest {
             Collection actualPlugins = TerraformEnvironmentStage.getPlugins()
             assertThat(actualPlugins, hasItem(instanceOf(ParameterStoreBuildWrapperPlugin.class)))
         }
+
+        @Test
+        void modifiesTerraformEnvironmentStageCommandWithGlobalParameter() {
+            ParameterStoreBuildWrapperPlugin.withGlobalParameter('/somePath/').init()
+
+            Collection actualPlugins = TerraformEnvironmentStage.getPlugins()
+            assertThat(actualPlugins, hasItem(instanceOf(ParameterStoreBuildWrapperPlugin.class)))
+        }
```

Comment:
`class Init` is meant to indicate that we're only unit testing the `.init()` method.  The definition of `init()` is 

https://github.com/manheim/terraform-pipeline/blob/6e2272d1fbeda7701d92a4ad480594bf55ef3ba9/src/ParameterStoreBuildWrapperPlugin.groovy#L10-L13

`init()` modifies the TerraformValidateStage, and the TerraformEnvironmentStage.  Those two behaviors are covered by the existing `modifiesTerraformEnvironmentStageCommand` test, and the new `modifiesTerraformValidateStageCommand`.  

I would not mix `withGlobablParameter` into the `class Init`, since `withGlobalParameter` is not called as part of init, and is not expected to be part of `init()`s behavior.  

I would move or remove the above test.

Reply:
removed